src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -16,7 +17,8 @@
 JOBS_NOT_PROVIDED_ERR = "job_id_list not provided"
 CELLS_NOT_PROVIDED_ERR = "cell_id_list not provided"
 BATCH_NOT_PROVIDED_ERR = "batch_id not provided"
-ONE_INPUT_TYPE_ONLY_ERR = "Please provide one of job_id, job_id_list, or batch_id"
+NO_INPUT_TYPE_ERR = "Please provide one of job_id, job_id_list, or batch_id"
+ONE_INPUT_TYPE_ONLY_ERR = "Please provide at most one of job_id, job_id_list, or batch_id"
```

Comment:
Thought this was a real distinction but now I'm not so sure with the reconfigured request types ...

Reply:
The frontend can't do anything useful with this info so really all these errors could be replaced by `NO_INPUT_TYPE_ERR`.

Reply:
Sure